app/helpers/events_helper.rb

```diff
@@ -16,4 +16,27 @@ def users_for_event(state)
     req = req.where(state: state) if state != 'all'
     user_email = req.map { |e| e.user.email }.uniq
   end
+
+  def check_states(requests)
+    if requests.map { |r| r.event = @event }.count > 1
```

Comment:
Hm, this is not nice! You could do sth like this ```requests.where(event: @event).count``` but that would still trigger n + 1 queries. @bgeuken any better solution? We could also iterate over the requests in the view but I think this is strange because the table is about users and not about requests and then we would need to do always request.user.foo to access attributes ...